index.html

```diff
@@ -1,3 +1,7 @@
+<!-- Add your HTML markup here -->
```

Comment:
No need to leave these comments behind! In fact, all non-relevant comments should not make it to the final PR.